### ISSUE TITLE
Add a code of conduct

### ISSUE THREAD

**Issue by blanchardjeremy** [microcovid/microcovid#545]
While reviewing the GitHub repo [Guide to Allyship](https://github.com/almnt/guide-to-allyship), I noticed they include a code of conduct to help create an inclusive community.

https://github.com/almnt/guide-to-allyship/blob/master/CODE_OF_CONDUCT.md

Looks like they borrowed it from here: https://www.contributor-covenant.org/version/2/0/code_of_conduct/

Curious what folks think.

**Comment by comradekingu** [comment]
@blanchardjeremy
I am just an irrelevant contributor, but wouldn't let that pass if I wasn't. I respect it isn't my call.
I wouldn't have been a contributor if it was there to be accepted beforehand, but in my irrelevancy here I am.
As far as meritocracy goes, (where that text-file is an attempt at dismantling it) my reasoning is:

It does for the benefit of a community, nothing other than lay out empty promises, which are no different being written, nor is doing something about it a premise changed in the slightest once stated.
There already is a CoC for GitHub, but nobody reads that either.
The Covenant CoC allows invariant sections, which means it has to be read meticulously for every project, and each time it changes.
Continual reiteration of groups to be effectively singled out is counterproductive, and does not amount to exposure-therapy. It is actually catastrophizing and creates (paradoxically), the divide it supposedly seeks to do away with in so doing. In all respects, the opposite of cognitive behavioral therapy. (Were it not, it would alert the reader to problems that just aren't there; where it would otherwise warrant its inclusion. I should hope not for the case of microCOVID.)
Where it is redundant to law it takes on an illusion of self-importance. It never was in a position to allow harassment, for example.
For anyone that does read it, a false sense of security is gained, because most people don't.
It works on the premise of supposedly thwarting anti-social behavior by
adding "disregard a text-file" to what is effectively a greater set of ingressions.
It is deeply anti-human in doing away with sexuality, and it favours professionalism over community.
It is deeply authoritarian, followed up with the vaguest of rationale for using ultimate power.
Where it isn't vague, it is endless bloat trying to navigate social hierarchy by way of writing unwritten rules.
It comes in a plethora of versions for this reason.
The 2.0 version has a section on social media, and replaces (from 1.4) ~"maintainers" with "Community leaders".
The power utilized is a secret operation that does away with a trifecta of power, no facing ones accuser, and it escalates personal conflicts beyond the control of the individuals involved.
This way of doing it is an obvious way to get people that are the most unfit for the position into the place they can do the most harm, without having made any sacrifices or sinking any personal debt into the project.
It is now longer than ever, meaning read less seldom, with less of its total remembered if so.
It has nothing but major fiascos in its wake, and it takes up a whole lot of productive time in non-productive discussions.
The various CoCs are mutually exclusive, so the premise of saying there is one is near meaningless.
The societies that champion it are, if I am to be completely neutral here, _at-least_ no better than their counterparts.
It is most often instated without any discussion so GitHub projects can have a green checkmark, (often without maintainers clicking buttons having read it).

One will find it is perfectly possible to have a community without it, as microCOVID was, and that contributors are not asked to pledge allegiance to this text-file, making even that part hollow.

Its history in GitHub (or CoCs before then) etc. is nothing short of absolute satire, but that is another (arguably anecdotal) matter.

The only good part (IMO), which curiously _isn't_ a CoC (and isn't novel) is the contact field. As for the community of microCOVID translators on Weblate, the project dropped the Covenant CoC in favour of https://github.com/WeblateOrg/weblate/blob/master/CONTACT.md with zero problems. Similar to microCOVID, it didn't have one when it was established, which speaks volumes about its efficacy or supposed protection of community-building.

For all my Mozilla contributions and insight, it is a failing company and has been for some time. To the credit of having CoC's however, Mozilla requires projects to have one to be legible for their funding. (Last that wast the case last time I applied for a grant.)

TL;DR Most anyone can do a whole lot in a project, and a CoC doesn't change that for the better.
As an exercise in why objective language matters, do tell me I am not an "ally", for a meaningful definition of the word as it relates to community.
